posts/2024-02-01-january-ecodev.mdx

```diff
@@ -0,0 +1,123 @@
+
+**ETHLatam 13th - 14th March**
+
+Waku has $3,500 in bounties on offer during the hackathon. The team on the ground will support the hackers across the two days in building their projects for submission, and we expect to see many more new use cases for Waku.
+
+Anybody hosting a side event at ETHDenver or ETHLatam and would like to collaborate with Waku, drop us message on [Discord](https://discord.com/invite/gMPAzmcDER). 
+
+
+## Press and Media
+
+**Vitalik Buterin Recognises Waku: **This month, Waku received an excellent shout-out from Vitalik Buterin, who recognised Waku as the successor of Whisper, the original peer-to-peer communications protocol for Ethereum. In a [blog post](https://vitalik.eth.limo/general/2023/12/28/cypherpunk.html#ethereum-as-part-of-a-broader-technological-vision), Buterin supported Waku as the realisation of the cypherpunk messaging protocol imagined by Gavin Wood.
+
+The story was also covered in [CoinTelegraph](https://cointelegraph.com/press-releases/vitalik-buterin-recognises-waku-as-the-peer-to-peer-communications-layer-of-gavin-woods-web3-vision), [CryptoDaily](https://cryptodaily.co.uk/2024/01/meet-waku-the-peer-to-peer-communications-protocol-recognized-by-vitalik-buterin), and [The Defiant](https://thedefiant.io/waku-the-p2p-communication-protocol-for-web3-recognized-by-vitalik-buterin). 
+
+**Waku was voted as a top privacy project of 2023**: Web3 Privacy Now included Waku on their long list of [top privacy-focused projects](https://mirror.xyz/0x0f1F3DAf416B74DB3DE55Eb4D7513a80F4841073/LdOt0mlG_nXqhnmLf7s-X13NDNdpwQw72ESKaJq-0zA) of the year. The finalists were voted on by over 100 privacy enthusiasts, developers, researchers, cryptographers, cypherpunks, and community builders from DarkFi, Nym, Swarm, Logos, Railgun, and many other organisations.
+
+
+## Technical Releases
+
+**The Waku React package:** The package provides the components and UI adapters needed to integrate Waku SDK into React applications effortlessly. We developed the package in response to feedback from hackathon participants last year. The documentation, including detailed steps on using the package, [can be found here](https://docs.waku.org/guides/js-waku/use-waku-react/). 
+
+There is also a [beginner-friendly tutorial](https://blog.waku.org/tictactoe-tutorial/) for building a simple tic-tac-toe game that can be played by two players who are connected using Waku.
+
+**Open Bounties:** We have reviewed and revised the Waku bounties, ensuring that these tasks are beginner-friendly. Our goal is to make it easier for newcomers to Waku to contribute and, in turn, support the growth of the ecosystem.Anyone can work on the bounties at their own pace, as there is no hard cut-off date. When a bounty proposal is approved, it will be marked as ‘Assigned’. Find the open bountie [here.](https://github.com/waku-org/bounties/issues)
```

Comment:
```suggestion
**Open Bounties:** We have reviewed and revised the Waku bounties, ensuring that these tasks are beginner-friendly. Our goal is to make it easier for newcomers to Waku to contribute and, in turn, support the growth of the ecosystem. Anyone can work on the bounties at their own pace, as there is no hard cut-off date. When a bounty proposal is approved, it will be marked as ‘Assigned’. Find the open bountie [here.](https://github.com/waku-org/bounties/issues)
```